practices/playbook.md

```diff
@@ -0,0 +1,101 @@
+---
+title: The Practice Playbook
+description: How do you run a practice at Artsy?
+---
+
+# Practice Playbook
+
+## Establish Goals For The Practice
+
+- It should be clear why the practice is meeting and why it is a good use of time
+  - Is there a cross team concern that needs alignment?
+  - What does the future of Artsy's stack look like from the perspective of the practice? Is this practice how we
+    get there?
+- It is ok (and maybe even preferred) for these goals to be short and concrete. Later on, the practice's goals may
+  evolve into a technical vision/roadmap.
+
+## Create A Landing Page In Notion
+
+This represents a single source of truth for the practice group.
+
+### Create A Meeting Agenda
+
+Creating an agenda will keep the practice grounded and consistent. An example agenda might be:
+
+1. Ask who is taking notes for this meeting?
+2. Ask for updates on any action items from previous meetings
+3. Go over any metrics related to the practice
+4. Ask for updates from any working groups or tasks forces that have overlapping concerns with the practice.
+5. Decide on a topic for next week
+6. Discuss this week's topic
+
+### Create Backlog Of Topics
+
+Practice members should be encouraged to add ideas and topics to the backlog. It's important to note that a member
+doesn't have to be the expert to request a topic to be presented. Practices can be another venue for knowledge
+sharing at Artsy.
+
+Store these topics in a table on the practice's Notion page.
+
+### Create Template Or Playbook For Discussion Topics
+
+It is useful for discussion topics to have the following:
+
+1. Title: a few words describing the topic
+2. Champion: a designated person driving the discussion, doesn't have to be the expert here!
+3. Type of discussion: tech plan review, brainstorm, knowledge share, etc
+4. Goal of discussion: what questions should be answered? what kind of feedback does the champion hope to get?
+
+See the Platform Practice's
+[Discussion Topic Playbook](https://www.notion.so/artsy/Discussion-Topic-Playbook-4a11a9c2313c409782b3f33eda9bb4ee)
+
+### Create Jira Board To Track Action Items
+
+Action items from the practice meeting should be tracked on product teams' projects but visible in one aggregated
+board. The practice should not have a separate board to track work. Action items from practices should ideally be
+folded into a normal sprint on a product team.
+
+- Create a new board
+- Create a new label to mark tickets as related to the practice
+- Change the board settings to filter by label and remove project filter
```

Comment:
I struggled with these steps a bit ([here](https://artsy.slack.com/archives/C8VH98J6Q/p1572622567002000)). You have to label an issue with the new practice label first, then create a "saved filter query" targeting that label, then create a new board in an existing project (choose the most relevant) and associate it with the saved filter query.

Reply:
thanks! i'll try to fold those steps in clearly